group.go

```diff
@@ -174,6 +176,59 @@ func (cli *Client) UpdateGroupParticipants(jid types.JID, participantChanges map
 	return resp, nil
 }
 
+// GroupRequestParticipantsList get the list of participants waiting to join the specified group in WhatsApp.
+func (cli *Client) GroupRequestParticipantsList(jid types.JID) []*types.JID {
+	res, err := cli.sendGroupIQ(context.TODO(), iqGet, jid, waBinary.Node{
+		Tag: "membership_approval_requests",
+	})
+	if err != nil {
+		fmt.Printf("Possibly failed to sendGroupIQ request for GroupRequestParticipantsList: %+v", err)
+		return nil
+	}
+	participantsnode, ok := res.GetOptionalChildByTag("membership_approval_requests")
+	if !ok {
+		return nil
+	}
+	children := participantsnode.GetChildren()
+	participants := make([]*types.JID, 0, len(children))
+	for _, child := range children {
+		childAG := child.AttrGetter()
+		switch child.Tag {
+		case "membership_approval_request":
+			part := childAG.JID("jid")
+			if !childAG.OK() {
+				fmt.Printf("Possibly failed to parse %s element in membership_approval_request node: %+v", child.Tag, childAG.Errors)
+			}
+			participants = append(participants, &part)
+		}
+	}
+	return participants
+}
+
+// UpdateGroupRequestParticipants can be used to accept or reject requests to join WhatsApp groups.
+func (cli *Client) UpdateGroupRequestParticipants(jid types.JID, participantChanges map[types.JID]ParticipantChange) (*waBinary.Node, error) {
+	content := make([]waBinary.Node, len(participantChanges))
+	i := 0
+	for participantJID, change := range participantChanges {
+		content[i] = waBinary.Node{
+			Tag: string(change),
+			Content: []waBinary.Node{{
```

Comment:
Someone reported that this was the wrong way for `UpdateGroupParticipants`, did you test this with multiple users?

Apparently the correct way is

```xml
<promote>
  <participant>...</participant>
  <participant>...</participant>
</promote>
```

instead of

```xml
<promote><participant>...</participant></promote>
<promote><participant>...</participant></promote>
```

If that is the case here too, I think the parameter for this function should be two lists of JIDs: one for approve and one for reject (I assume nothing other than approving/rejecting can be done here?)